ddns-ipv64.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>ddns-ipv64</Name>
+  <Repository>alcapone1933/ddns-ipv64:latest</Repository>
+  <Registry>https://hub.docker.com/r/alcapone1933/ddns-ipv64</Registry>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Project>https://github.com/alcapone1933/docker-ddns-ipv64</Project>
+  <Overview>This Docker container is a DDNS updater for Free DynDNS - ipv64.net.&#xD;
+&#xD;
+If the ipv4 address at the site is changed, the new ipv4 address is sent to ipv64.net as an A-Record.</Overview>
+  <Category>Tools: Network:DNS</Category>
+  <Icon>https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQasFPd0QKbbYpPvuUG0EXhoVFF-Kw2XcdqDg&amp;amp;s</Icon>
+  <Requires>List of all Parameters that can be configured https://github.com/alcapone1933/docker-ddns-ipv64?tab=readme-ov-file#env-parameter</Requires>
+  <Config Name="Timezone" Target="TZ" Default="Europe/Berlin" Mode="" Description="Timezone" Type="Variable" Display="always" Required="true" Mask="false"/>
+  <Config Name="Polling frequency IP address" Target="CRON_TIME" Default="*/15 * * * *" Mode="" Description="Polling frequency of the current IP address" Type="Variable" Display="always" Required="true" Mask="false"/>
+  <Config Name="Polling frequenz Domain" Target="CRON_TIME_DIG" Default="*/30 * * * *" Mode="" Description="Polling frequenz for the domain (dig DOMAIN_IPV64 A)" Type="Variable" Display="always" Required="true" Mask="false"/>
+  <Config Name="Domain(s) ipv64" Target="DOMAIN_IPV64" Default="" Mode="" Description="YOUR DOMAINS: e.g. your-domain.ipv64.net, your-domain.ipv64.de" Type="Variable" Display="always" Required="true" Mask="false"/>
+  <Config Name="Domain Key" Target="DOMAIN_KEY" Default="" Mode="" Description="DOMAIN KEY: YOUR DOMAIN KEY or DynDNS update hash can be found at https://ipv64.net/dyndns.php" Type="Variable" Display="always" Required="true" Mask="true"/>
+  <Config Name="Domain Präfix yes|no" Target="DOMAIN_PRAEFIX_YES" Default="yes|no" Mode="" Description="DOMAIN PRAEFIX YES: This means that the domain PRAEFIX is actively used" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="Domain Präfix" Target="DOMAIN_PRAEFIX" Default="" Mode="" Description="YOUR DOMAIN PRAEFIX (subdomain) : &#x26A0;&#xFE0F; Use only one prefix &#x26A0;&#xFE0F; e.g. ddns" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="IP check" Target="IP_CHECK" Default="yes|no" Mode="" Description="IP CHECK: The IP address of the domain is checked" Type="Variable" Display="always" Required="false" Mask="false"/>
```

Comment:
Fields with a dropdown picker should be marked Required, with the default action first in the pipe list.

```suggestion
  <Config Name="IP check" Target="IP_CHECK" Default="yes|no" Mode="" Description="IP CHECK: The IP address of the domain is checked" Type="Variable" Display="always" Required="true" Mask="false"/>
```